challenges/weekend2/weekend2:brainix.py

```diff
@@ -0,0 +1,107 @@
+'''Redis DEVcember weekend 2.
+
+When I ran this Python script, I got the following output:
+
+RedisSet cardinality: 5000
+BloomFilter count: 4997
+HyperLogLog count: 4974
+
+RedisSet hits: 5000, misses: 0, time: 356 ms
+BloomFilter hits: 5000, misses: 0, time: 1315 ms
+HyperLogLog hits: 3754, misses: 1246, time: 50 ms
```

Comment:
Question for code reviewers: I know that HyperLogLogs aren't designed for membership testing, but that's it's possible with a `PFADD` hack. Is it expected for HLL membership testing to be this inaccurate? Or is there a problem with my approach or with my code?